package.json

```diff
@@ -29,7 +29,7 @@
   "license": "SEE LICENSE IN README.md",
   "dependencies": {
     "@babel/runtime": "^7.12.1",
-    "@devextreme/runtime": "3.0.0",
+    "@devextreme/runtime": "~3.0.0",
```

Comment:
@dxvladislavvolkov I wonder if this change is okay: can I use non-strict version here?

Reply:
I think it is ok